src/components/GalleryPage/Gallery.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { graphql, useStaticQuery } from 'gatsby';
+import Img from 'gatsby-image';
+import Container from '@material-ui/core/Container';
+import Slider from 'react-slick';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import './gallery.css';
+
+export default function Gallery() {
+	const settings = {
+		dots: true,
+		infinite: true,
+		speed: 500,
+		slidesToShow: 1,
+		slidesToScroll: 1,
+		className: 'slides'
+	};
+
+	const data = useStaticQuery(graphql`
+		{
+			carouselImages: allFile(filter: {relativePath: {glob:"carousel/*"}}) {
+				nodes {
+					id
+					relativePath
+					childImageSharp {
+						fluid {
+							...GatsbyImageSharpFluid
+						}
+					}
+				}
+			}
+		}
+	`);
+
+	const image= []
```

Comment:
This is an array variable; let's make the name plural to indicate that, for readability.

```suggestion
	const images = [];
```